vkcs/resource_vkcs_publicdns_record.go

```diff
@@ -0,0 +1,336 @@
+package vkcs
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+const (
+	recordDelay         = 5 * time.Second
```

Comment:
why we do not use default values?

Reply:
Updated (if you meant these ones):
 https://github.com/vk-cs/terraform-provider-vkcs/blob/2034e53dc9e98eaf6f85c39fed948e94dd9bb44a/vkcs/resource_vkcs_publicdns_record.go#L16-L21

Reply:
i thought you overriden default values. but thera are no such ones